extension/agenthealth/handler/stats/agent/agent.go

```diff
@@ -76,6 +77,9 @@ func (s *Stats) Merge(other Stats) {
 	if other.Mode != nil {
 		s.Mode = other.Mode
 	}
+	if other.EntityRejected != nil {
+		s.EntityRejected = other.EntityRejected
```

Comment:
have we added this to the agent stats handler yet? I assume its TBD if not?

Reply:
yes it's already added to the agent stats handler